### DISCUSSION TITLE
搜索建议会盖住输入法

### DISCUSSION THREAD

**Discussion by cest713** [ywmoyue/biliuwp-lite#147] [General]
如题，在搜索框中输入相关内容时，搜索建议会盖在输入法的列表上面，导致看不到输入法提示的文字内容

**Comment by ywmoyue** [comment]
问下是什么设备什么版本的操作系统

我试了Win11 21h2正常桌面设备呼出键盘和Win10的SteamDeck触屏键盘都没出现搜索建议盖在键盘上面

**Comment by TPAXcc** [comment]
Windows.10.21390
新旧两种输入法都复现不了

> 问下是什么设备什么版本的操作系统
> 
> 我试了Win11 21h2正常桌面设备呼出键盘和Win10的SteamDeck触屏键盘都没出现搜索建议盖在键盘上面

**Comment by ywmoyue** [comment]
> 问下是什么设备什么版本的操作系统
> 
> 我试了Win11 21h2正常桌面设备呼出键盘和Win10的SteamDeck触屏键盘都没出现搜索建议盖在键盘上面

超过一个月未回复，当前Issue转为讨论
